app/src/main/java/au/com/bluedot/minimalintegration/AppTempoReceiver.java

```diff
@@ -0,0 +1,28 @@
+package au.com.bluedot.minimalintegration;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+import android.widget.Toast;
+import au.com.bluedot.point.net.engine.BDError;
+import au.com.bluedot.point.net.engine.TempoTrackingReceiver;
+import org.jetbrains.annotations.NotNull;
+
+public class AppTempoReceiver extends TempoTrackingReceiver {
+    private final Handler handler = new Handler(Looper.getMainLooper());
+
+    /**
```

Comment:
I'm not against adding the documentation in this project, but we should be consistent - why here and not in the geotriggerReceiver?